app/views/hyku_mailer/depositor_email.html.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    body { font-family: Arial, sans-serif; line-height: 1.6; }
+    .email-container { max-width: 600px; margin: auto; padding: 20px; border: 1px solid #ccc; }
+    .footer { font-size: 12px; color: #666; margin-top: 20px; }
+    .highlight { color: #0056b3; }
+    .button {
+      display: inline-block;
+      padding: 10px 20px;
+      margin: 10px 0;
+      background-color: #0056b3;
+      color: white;
+      text-decoration: none;
+      border-radius: 5px;
+    }
+  </style>
+</head>
+<body>
+  <div class="email-container">
+    <p>Dear Author,</p>
+    <p>You had <%= pluralize(@statistics[:new_file_downloads], 'download') %> in <%= Date.today.prev_month.strftime("%B %Y") %> across your <%= pluralize(@statistics[:new_work_views], 'work') %> in <%= @application_name %>. Your current readership:</p>
+    <ul>
+      <li><%= pluralize(@statistics[:total_file_downloads], 'Total File Download') %></li>
+      <li><%= pluralize(@statistics[:total_work_views], 'Total Work View') %></li>
+    </ul>
+    <a href="<%= @url %>" class="button">VISIT MY DASHBOARD</a>
+    <div class="footer">
+      <p>These monthly reports are provided to you by bepress on behalf of <%= @application_name %>. For questions, comments, or to add more content and increase your readership and visibility as an author, please contact your repository administrator.</p>
```

Comment:
what is bepress

Reply:
Looks like that was in the original template and was simply copy and pasted in.  https://bepress.com/